ddpui/api/client/airbyte_api.py

```diff
@@ -268,6 +269,9 @@ def get_airbyte_destination_definitions(request):
         raise HttpError(400, "create an airbyte workspace first")
 
     res = airbyte_service.get_destination_definitions(orguser.org.airbyte_workspace_id)
```

Comment:
why don't you use this directly?  org = request.orguser.org and then use org everywhere

Reply:
i could but it hardly makes a difference... readability is the same, i doubt there is a performance difference and even if there is this is not meant to be high-performant code. it's the kind of change that if someone else did it i would roll my eyes but not stop them